code/modules/research/designs/weapon_designs.dm

```diff
@@ -3,37 +3,33 @@
 /////////////////////////////////////////
 
 /datum/design/nuclear_gun
-	name = "Advanced Energy Gun"
-	desc = "An energy gun with an experimental miniaturized reactor."
+	name = "Advanced Energy Gun Parts Kit"
```

Comment:
I'm not entirely sure but I *think* you might want to add some \impropers here and there so it says 
"That's an Advanced Energy Gun Parts Kit"
```suggestion
	name = "\improper Advanced Energy Gun Parts Kit"
```

Reply:
needs to be set in the guncrafting.dm file since that's where the actual parts kits items are. These are just the recipes you see on the RND console. Besides that everything else should be applied